ubuntudesign/vanilla-framework
# Audit and refactor framework settings

## Done

- Remove global settings that are too wide ranging in scope
- Split out remaining variables into distinct categories based on function
- Rename all variables to prefix them with their category
- Update `docs` with changes
- Update Release Notes to document changes

## QA

* Sanity check code changes
* Pull down code and run `gulp build`
* Ensure build passes

Fixes: #386
